package.json

```diff
@@ -10,15 +10,18 @@
     "start": "./src/script/index.js",
     "test": "jest",
     "coveralls": "jest --coverage && cat ./coverage/lcov.info | coveralls",
-    "build:prod": "cross-env NODE_ENV=build rollup -c ./misc/rollup.config.js",
-    "build:dev": "npm run build:prod -- -w",
-    "build": "npm run build:prod"
+    "build:gui:prod": "cross-env NODE_ENV=build rollup -c ./misc/rollup.gui.config.js",
+    "build:gui:dev": "npm run build:gui:prod -- -w",
+    "build:script:prod": "cross-env NODE_ENV=build rollup -c ./misc/rollup.script.config.js",
+    "build:script:dev": "npm run build:script:prod -- -w",
```

Comment:
I've got a build error in watch mode for the script.
When I want to update a file, I have this error : 
```
🚨   Could not load fs (imported by /Users/zuhmilo/workspaces/workspace-perso/react-workbench/src/script/config/bluebird.js): ENOENT: no such file or
 directory, open 'fs'
```

I think we need a [rollup plugin](https://github.com/calvinmetcalf/rollup-plugin-node-builtins), or put the platform target in babel configuration to node